test/e2e/csc_tests.go

```diff
@@ -18,21 +21,22 @@ const (
 
 	// targetNamespace is the non-existing target namespace
 	targetNamespace string = "non-existing-namespace"
+
+	// nonExistingPackageCSCName is the name of the catalogsourceconfig that points
+	// to a non-existing Package
+	nonExistingPackageCSCName string = "non-existing-package-operators"
+
+	// packageName is the non-existing package
+	packageName string = "non-existing-package"
 )
 
-func runCSCWithNonExistingTargetNamespace(t *testing.T) {
+func runCSCTests(t *testing.T) {
 	ctx := test.NewTestCtx(t)
 	defer ctx.Cleanup()
 
 	// Get global framework variables
 	f := test.Global
-	// Run tests
-	if err := cscWithNonExistingTargetNamespace(t, f, ctx); err != nil {
-		t.Fatal(err)
-	}
-}
 
-func cscWithNonExistingTargetNamespace(t *testing.T, f *test.Framework, ctx *test.TestCtx) error {
 	// Get test namespace
 	namespace, err := ctx.GetNamespace()
```

Comment:
What is the benefit of adding the namespace check here? Each test also individually checks for the namespace as well.